src/main/java/uk/gov/companieshouse/chd/order/api/interceptor/ApiKeyAuthenticationInterceptor.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.chd.order.api.interceptor;
+
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+import uk.gov.companieshouse.api.util.security.AuthorisationUtil;
+import uk.gov.companieshouse.api.util.security.SecurityConstants;
+import uk.gov.companieshouse.logging.Logger;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static uk.gov.companieshouse.chd.order.api.logging.LoggingUtils.getLogger;
+
+public class ApiKeyAuthenticationInterceptor extends HandlerInterceptorAdapter {
+
+    private static final Logger LOGGER = getLogger();
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+
+        final String authorisedUser = AuthorisationUtil.getAuthorisedIdentity(request);
+        if (authorisedUser == null) {
+            LOGGER.debugRequest(request, "no authorised identity", null);
```

Comment:
Should these `debugRequest()` calls be `infoRequest()` calls?

Reply:
Yes I will change this